src/helix-anon/index.html

```diff
@@ -93,7 +93,8 @@
             return { 'IsHelixWorkItem': false };
         }
         try {
-            var data = JSON.parse(comment);
+            var escapedComment = comment.replaceAll("$quot;", "\"")
+            var data = JSON.parse(escapedComment);
```

Comment:
Is `$quot;` correct❔ I expected `&quot;`.

minor but
- Why not reuse the existing variable e.g., `comment = comment.replaceAll(...);`❔
- The `escapedComment` name is confusing because you're **un**escaping
- We could be more specific focusing on strings like ` &quot;`, `&quot;:`, and `&quot;,` i.e., the places quotation marks are important to JSON

Reply:
You're right, this is incorrect. I'll apply your feedback to a followup PR. The validation story for this repo is atrocious so every manual check I did didn't work out that well.